library/rsa.c

```diff
@@ -493,14 +493,17 @@ size_t mbedtls_rsa_get_len( const mbedtls_rsa_context *ctx )
 
 /*
  * Generate an RSA keypair
+ *
+ * This generation method conforms to FIPS 186-4 if 2^16 < exponent < 2^256 and
```

Comment:
Same comment as above: I'd prefer a less strong wording, not claiming FIPS compliance.